tests/media/server/ipc/sessionManagementServer/SessionManagementServerTestsFixture.cpp

```diff
@@ -184,12 +186,57 @@ void SessionManagementServerTests::serverWillSetLogLevels()
 
 void SessionManagementServerTests::sendServerInitialize()
 {
-    EXPECT_TRUE(m_sut->initialize(socketName, socketPermissions, socketOwner, socketGroup));
+    std::unique_ptr<firebolt::rialto::common::ILinuxWrapper> linuxWrapper =
+        firebolt::rialto::common::ILinuxWrapperFactory::createFactory()->createLinuxWrapper();
+    EXPECT_TRUE(m_sut->initialize(linuxWrapper, socketName, socketPermissions, socketOwner, socketGroup));
 }
 
 void SessionManagementServerTests::sendServerInitializeAndExpectFailure()
 {
-    EXPECT_FALSE(m_sut->initialize(socketName, socketPermissions, socketOwner, socketGroup));
+    std::unique_ptr<firebolt::rialto::common::ILinuxWrapper> linuxWrapper =
+        firebolt::rialto::common::ILinuxWrapperFactory::createFactory()->createLinuxWrapper();
+    EXPECT_FALSE(m_sut->initialize(linuxWrapper, socketName, socketPermissions, socketOwner, socketGroup));
+}
+
+void SessionManagementServerTests::testSocketOwnership(bool testOwnerIsValid, bool testGroupIsValid)
```

Comment:
Split this method up, methods should have minimal logic so that the test can be followed clearly.